src/inquirer/render/console/_editor.py

```diff
@@ -28,6 +27,17 @@ def process_input(self, pressed):
             raise KeyboardInterrupt()
 
         if pressed in (key.CR, key.LF, key.ENTER):
+            # The import of python-editor needs to be here, at this
+            # low level to prevent 'editor' itself from importing
+            # distutils.spawn until it is actually being used. This
+            # has to do with ubuntu (debian) python packages
+            # artificially separated from distutils.
+            #
+            # If this import is at top level inquirer breaks on ubuntu
+            # until the user explicitly apt-get install
+            # python3-distutils. With the import here it will only
+            # break if the code is utilizing the Editor prompt.
+            import editor
             data = editor.edit(contents=self.question.default or "")
```

Comment:
Could you please provide a test for CI/CD that would not work if the import is in its original place?